tests/testthat/test-epi_slide.R

```diff
@@ -4,31 +4,55 @@ edf = dplyr::bind_rows(
   dplyr::tibble(geo_value = "ak", time_value = as.Date("2020-01-01") + 1:200, value=1:200),
   dplyr::tibble(geo_value = "al", time_value=as.Date("2020-01-01") + 1:5, value=-(1:5))
 ) %>%
-  as_epi_df()
+  as_epi_df() %>%
+  group_by(geo_value)
 
 f = function(x, ...) dplyr::tibble(value=mean(x$value), count=length(x$value))
 
 ## --- These cases generate the error: ---
+test_that("`after` must be defined as a non-zero integer if `before` is missing", {
+  expect_error(epi_slide(edf, f, after = 0L, ref_time_values=as.Date("2020-01-01")),
+               "`before` cannot be missing when `after` is set to 0.")
+})
```

Comment:
Also explicitly check the case where the user doesn't specify `after` at all.   (I assume that this triggers the same error as the above test given the current implementation.  But we want to ensure that it remains this way in the future.)  We should also check that the error message is helpful for debugging in this case.